src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -554,19 +564,19 @@ const StyledSelectedIndicator = styled.div<StyledSelectedIndicatorProps>`
     css`
       bottom: 0px;
       left: 0px;
+      right: 0px;
```

Comment:
**question**: these again seem to be changes that aren't just for new focus styling (here and the two removed styles a little further down)

Reply:
It affects how the new styling looks, this is just mimicking the width/ height that was previously, there's no visual regression by doing this afaik